experimental/examples/attributable-map-demo/package.json

```diff
@@ -0,0 +1,84 @@
+{
+	"name": "@fluid-example/attributable-map-demo",
+	"version": "2.0.0-internal.4.1.0",
+	"private": true,
+	"description": "An web app demo using the attributableMap",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "experimental/examples/attributable-map-demo"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "lib/index.js",
+	"module": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"scripts": {
+		"build": "react-scripts build",
+		"eject": "react-scripts eject",
+		"format": "npm run prettier:fix",
+		"lint": "npm run prettier",
+		"lint:fix": "npm run prettier:fix",
+		"prettier": "prettier --check . --ignore-path ./.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ./.prettierignore",
+		"start": "react-scripts start",
```

Comment:
Oh I see, this is from create-react-app.  Consider aligning with the startup behaviors of other demos - this can make it cheaper for other engineers in the future to make horizontal changes that touch all demos.